### PR TITLE
Proposed fix for issue #294 (updated)

### PR DESCRIPTION
**Pull request by Cris70**
As per the subject.

### CODE REVIEW

File: usr/lib/webapp-manager/common.py [hunk 1]
```diff
@@ -246,7 +250,7 @@ def create_webapp(self, name, url, icon, category, browser, custom_parameters, i
             desktop_file.write("[Desktop Entry]\n")
             desktop_file.write("Version=1.0\n")
             desktop_file.write("Name=%s\n" % name)
-            desktop_file.write("Comment=%s\n" % _("Web App"))
+            desktop_file.write("Comment=%s\n" % gettext.gettext("Web App"))
```
Comment (mtwebster) [inline usr/lib/webapp-manager/common.py:253]: This shouldn't be necessary:
https://github.com//linuxmint/webapp-manager/blob/master/usr/lib/webapp-manager/common.py#L50

**Comment by mtwebster** [comment]
In the future, please have a description here, instead of just referring to the issue report.

You provide a good amount of detail there, but here should be 'what was wrong', 'what I did' and 'why' .. like:

```
(PR title:) Fix profile creation for Falkon browser
```
```
Falkon's command line arguments were incorrect and causing it to
create a new profile in ~/.config.

- Fixed generated command line
- Symlink to the ~/.config profile location for backward compatibility
```

Ideally something like this would have a single commit, and the above would also be in the commit message.

No big deal here, it just makes it easier to look at history without unnecessarily having to visit pull requests and/or original issues.

Thanks!

**Comment by anaximeno** [comment]
@mtwebster I suggest adding something like this (with the best practice for commits on the Linux Mint projects) on the Linux Mint Developer guide, I believe it isn't there if I remember correctly.

**Comment by Cris70** [comment]
> In the future, please have a description here, instead of just referring to the issue report.

Thank you Micheal, will do!

**Pull request merged**